src/java/org/apache/cassandra/index/sai/disk/v1/postings/PostingsReader.java

```diff
@@ -318,7 +313,7 @@ private int nextFoRValue()
         }
```

Comment:
So above on 310...

Given the change below, replacing `null` w/ `LongValues.ZEROES`, do we even need `if/else` here, or does the logic just work without it?

Reply:
good catch, fixed